File: rest/accounts/instance-post-example-1/instance-post-example-1.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Api.V2010;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/console
        // To set up environmental variables, see http://twil.io/secure
        const string accountSid = Environment.GetEnvironmentVariable("TWILIO_ACCOUNT_SID");
        const string accountSidToSuspend = Environment.GetEnvironmentVariable("TWILIO_SUBACCOUNT_SID");
        const string authToken = Environment.GetEnvironmentVariable("TWILIO_AUTH_TOKEN");

        TwilioClient.Init(accountSid, authToken);

        AccountResource.Update(
            pathSid: accountSidToSuspend,
            status: AccountResource.StatusEnum.Suspended);
    }
}

File: rest/accounts/instance-post-example-1/instance-post-example-1.5.x.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once '/path/to/vendor/autoload.php'; // Loads the library
use \Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

// Get an object from its sid. If you do not have a sid,
// check out the list resource examples on this page
$account = $client
    ->accounts
    ->getContext("ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
    ->update(
        array('status' => 'suspended')
    );

echo $account->dateCreated->format('Y-m-d H:i:s');